packages/breadcrumbs/src/index.js

```diff
@@ -6,6 +6,6 @@
  */
 
 export { default as Breadcrumb } from './elements/Breadcrumb';
-export { default as BreadcrumbView } from './views/BreadcrumbView';
-export { default as List } from './views/List';
-export { default as Item } from './views/Item';
+export { default as BreadcrumbView } from './styled/BreadcrumbView';
```

Comment:
We no longer export any of the `styled` elements to help reduce API churn. Let's remove `BreadcrumbView`, `List`, and `Item.

Reply:
Also, looks like this file should still be moved to `TS`. This would correct some naming errors here as well.

Reply:
Oh these comments were blazing fast! I saw this during my second review and addressed with new commit. I'll go ahead and address other comments too. I'll re-request a review once resolve them 👍 .